07-final-project/components/character-info-list.js

```diff
@@ -0,0 +1,29 @@
+var React = require('react');
+
+var CharacterInfoList = React.createClass({
+
+    render: function () {
+
+        return (
+            <div>
+                    {this.renderCharacterInfo()}
+            </div>
+        );
+    },
+
+    renderCharacterInfo: function () {
```

Comment:
Maybe there's no need to abstract this to a `renderCharacterInfo` method (it's OK like it is, but maybe you want to define a method outside your main `render` when you have some logic to do and in this case we're just displaying info).